src/devices/depthCamera/depthCameraDriver.cpp

```diff
@@ -100,36 +112,29 @@ void streamFrameListener::onNewFrame(openni::VideoStream& stream)
 }
 
 
-depthCameraDriver::depthCameraDriver() : m_depthFrame(NULL), m_imageFrame(NULL), m_cameraDescription(NULL)
+depthCameraDriver::depthCameraDriver() : m_depthFrame(NULL), m_imageFrame(NULL), m_paramParser(NULL)
```

Comment:
You might consider using `nullptr` in place of `NULL` throughout the code.
Of course, this would be only optional in case you wanted to adhere to the already existing code.